app/components/mandatarissen/mandataris-card.js

```diff
@@ -51,4 +59,24 @@ export default class MandatarisCardComponent extends Component {
   stopEditingStatus() {
     this.editingStatus = false;
   }
+
+  addReplacement = task(async (replacement) => {
+    const newMandatarisProps = await this.mandatarisService.createNewProps(
+      this.args.mandataris,
+      {
+        start: new Date(),
+        publicationStatus: await getDraftPublicationStatus(this.store),
+      }
+    );
+    const lidmaatschap = await this.args.mandataris.heeftLidmaatschap;
+    const replacementMandataris =
+      await this.mandatarisService.getOrCreateReplacement(
+        this.args.mandataris,
+        replacement,
+        newMandatarisProps,
+        await lidmaatschap.binnenFractie
```

Comment:
I think we really need to rethink these fracties of replacements, but could be a refinement story or something like that?
I think it goes wrong here somewhere if the person you are trying to replace has no fractie:
![Screenshot 2024-09-20 at 10 29 07](https://github.com/user-attachments/assets/1b405bdf-a241-4d4f-9658-3ddf28b66d82)

Reply:
Very true did have that issue to! We should flag or limit functionality when not all information is set.